packages/app/src/lib/CustomTechRadar.ts

```diff
@@ -2,12 +2,24 @@ import {
   TechRadarApi,
   type TechRadarLoaderResponse,
 } from '@backstage/plugin-tech-radar';
+import { JanusBackstageCustomizeApi } from '../api';
 
 export class CustomTechRadar implements TechRadarApi {
+  private readonly janusBackstageCustomizeApi: JanusBackstageCustomizeApi;
+  constructor(options: {
+    janusBackstageCustomizeApi: JanusBackstageCustomizeApi;
+  }) {
+    this.janusBackstageCustomizeApi = options.janusBackstageCustomizeApi;
+  }
   async load(id: string | undefined): Promise<TechRadarLoaderResponse> {
-    const data = (await fetch(`/tech-radar/data-${id}.json`).then(res =>
-      res.json(),
-    )) as TechRadarLoaderResponse;
+    let data;
+    try {
+      data = await this.janusBackstageCustomizeApi.getTechRadarDataJson();
+    } catch (e) {
+      data = (await fetch(`/tech-radar/data-${id}.json`).then(res =>
```

Comment:
Could this be just done with awaits only?

Reply:
we can do that but then needs to await i.e 

```
const res = await fetch(`/tech-radar/data-${id}.json`) // as this return promise
data = await res.json();
```

LMK, your thoughts